addons/mail/static/src/new/thread/message.js

```diff
@@ -176,12 +176,12 @@ export class Message extends Component {
         if (this.message.isSelfAuthored) {
             return false;
         }
-        return this.props.thread.chatPartnerId !== this.message.author.partner?.id;
+        return this.props.thread.chatPartnerId !== this.message.author.id;
```

Comment:
I don't want to make code harder to maintain and read, and since the bug is non-critical, it's alright to keep this problem for a very short amount of time.

I will do follow-up PR to convert `chatPartnerId` to `chatPersona`, so that comparison is on record, which will fixes the issue immediately